config/webpack.config.js

```diff
@@ -308,7 +303,7 @@ module.exports = async (env = {}) => {
             {
               loader: 'css-loader',
               options: {
-                sourceMap: enableCSSSourcemaps,
+                sourceMap: true,
```

Comment:
I think we had this disabled for localhost builds because of spikes in memory usage when running `yarn watch`. I noticed memory usage without CSS source maps stays around ~3 GB for me, but when enabled jumps up to about 5 GB. Did you have similar results when running locally?

Reply:
no, nothing I noticed

Reply:
Might just be something for us to keep an eye out for